Examples/Tests/semi_implicit_electrostatic/analysis.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+sys.path.insert(1, "../../../../warpx/Regression/Checksum/")
+import checksumAPI
+
+filename = sys.argv[1]
```

Comment:
Could you add physics tests that checks that the expansion of the cylinder is as expected?
(and that would fail e.g. when using the regular explicit labframe solver instead of the semi-implicit labframe solver)